packages/server/src/modules/user/service.ts

```diff
@@ -127,6 +129,32 @@ export const findCoaches = (
   });
 };
 
+export const getPublicCoaches = async () => {
+  const coaches = await findCoaches({});
+  const shuffledCoaches = utils
+    .shuffleArray(coaches)
+    .slice(0, PUBLIC_COACH_NUMBER)
+    .map((coach: User) => ({
```

Comment:
Why are you doing this?

Reply:
Getting a limited number of coaches and then limiting the data sent as said in the task.
I chose 8 as the limit because on the landing we show groups of 3 + 3rd group has the show more button (3 + 3 + 2).

Reply:
Few things:
- modify findCoaches to be able to limit the number of returned coaches from DB
- you shouldn't do mapping, but rather change projection (the fields you're getting from DB)